src/audio_unit/split.rs

```diff
@@ -32,4 +33,12 @@ impl AudioUnit for Split {
 
         Ok(())
     }
+
+    fn handle_midi_messages(&mut self, midi_messages: &[MidiMessage<'static>]) -> Result<()> {
```

Comment:
I don't really have a mental model for what a split is (eg vs a pipeline) at the moment

Reply:
Pipeline:

```
input - a - b - output
```

Split

```
        a 
input <   > output
        b
```

It splits up and then recombines the audio signal so that you can run multiple units on it without them interfering with each other. (eg. the "delay" units are all using the original audio input rather than being chained off each other)